Source/WebCore/html/ColorInputType.cpp

```diff
@@ -156,11 +156,7 @@ bool ColorInputType::isMouseFocusable() const
 bool ColorInputType::isKeyboardFocusable(KeyboardEvent*) const
 {
     ASSERT(element());
-#if PLATFORM(IOS_FAMILY)
     return element()->isTextFormControlFocusable();
```

Comment:
Pre-existing but because you touch it last you get to fix it.
```suggestion
    return protectedElement()->isTextFormControlFocusable();
```